nginx-mainline.yaml

```diff
@@ -108,7 +111,8 @@ pipeline:
           --with-stream_realip_module \
           --with-stream_geoip_module=dynamic \
           --with-stream_ssl_preread_module
-  - runs: |
+  - working-directory: nginx-mainline
```

Comment:
What does the word 'mainline' mean to you?
I would have interpreted this as upstream/main .

maybe `nginx-git` or `nginx-checkout` or `git-checkout` ?

Reply:
AFAICT it refers to the version stream of the package. I was just trying to be consistent with the package naming scheme.

Reply:
i hadn't even noticed this was `nginx-mainline.yaml`.